InventoryIndexer/Indexer/IndexHandler.php

```diff
@@ -66,7 +67,7 @@ public function saveIndex(IndexName $indexName, \Traversable $documents, string
 
         $columns = [IndexStructure::SKU, IndexStructure::QUANTITY, IndexStructure::IS_SALABLE];
         foreach ($this->batch->getItems($documents, $this->batchSize) as $batchDocuments) {
-            $connection->insertArray($tableName, $columns, $batchDocuments);
+            $connection->insertArray($tableName, $columns, $batchDocuments, AdapterInterface::INSERT_IGNORE);
```

Comment:
In case of ignoring the data of the last threads will be ignored, and in case with update the data of all threads will be overwritten by last, it is no difference what data we lose. But I changed my fix according to your recommendation, you can check it.

Reply:
I agree with you, but it's better to store data from the latest SQL request.